cpm

```diff
@@ -248,13 +248,13 @@ _dnf() {
     remove)  $SUDO dnf remove "$@";;
     list)    dnf list --installed;;
     count)   rpm -qa | tot;;
-    update)  $SUDO dnf check-update;;
-    upgrade) $SUDO dnf check-update && $SUDO dnf distro-sync;;
+    update)  echo "DNF does not support this option.";;
```

Comment:
Actually it seems that `dnf` does support `check-update`, and it is the recommended command if the user just want to check if there is any:

https://docs.fedoraproject.org/en-US/quick-docs/dnf/

Follow up from #60

Reply:
Sure, but it does not have the same functionality as the update function of other package managers, so it would be misleading to use it here.